build.zig

```diff
@@ -114,8 +114,22 @@ const pcreSources = [_][]const u8{
     "src/pcre/pcre_version.c",
     "src/pcre/pcre_xclass.c",
 };
-const tests = [_][]const u8{ "src/buffer_test.c", "src/csv_test.c", "src/writer_test.c", "src/cli_test.c" };
-const testIncludes = [_][]const u8{ "src/buffer.c", "src/memory.c", "src/encoding.c", "src/csv.c", "src/writer.c", "src/cli.c" };
+const tests = [_][]const u8{
```

Comment:
Actually, you should just ignore this PR, as I discover that this parsing is never even used, and I remove it in 22fcc2c34b9dcc9e62e133797c9eddedcf9f6859